### PR TITLE
Serve Wiki Only on wiki.metakgp.org and metakgp.org

### PR DESCRIPTION
**Pull request by harshkhandeparkar**

### CODE REVIEW

**Comment by rajivharlalka** [comment]
Looks good to me

**Review by rajivharlalka — DISMISSED** [review]

**Review by rajivharlalka — APPROVED** [review]

File: mediawiki/LocalSettings.php
Comment (chirag-ghosh) [inline mediawiki/LocalSettings.php:1]: this change is related to this PR?

Comment (harshkhandeparkar) [reply mediawiki/LocalSettings.php:1]: I am not sure how this change got here; it should have already been merged. The current wiki deployment has this change.

Comment (chirag-ghosh) [reply mediawiki/LocalSettings.php:1]: interesting

**Review by chirag-ghosh — APPROVED** [review]

**Pull request merged**